UI/Selenium/Steps/SelectHearingSteps.cs

```diff
@@ -62,10 +68,12 @@ public void ThenIShouldAbleToOpenQuicklinkOnNewBrowser()
         [When(@"I click on copy joining by phone details to clipboard")]
         public void WhenIClickOnCopyJoiningByPhoneDetailsToClipboard()
         {
-            ExtensionMethods.MoveToElement(Driver, SelectHearingPage.Quicklinks, _scenarioContext);
-            ExtensionMethods.FindElementWithWait(Driver, SelectHearingPage.Phonetoclipboard, _scenarioContext).Click();
-            ApplicationData.hearingPhone = new TextCopy.Clipboard().GetText();
-            
+            if (ExtensionMethods.WaitForPageLoad(Driver, SelectHearingPage.Quicklinks, _scenarioContext))
+            {
+                ExtensionMethods.MoveToElement(Driver, SelectHearingPage.Quicklinks, _scenarioContext);
+                ExtensionMethods.FindElementWithWait(Driver, SelectHearingPage.Phonetoclipboard, _scenarioContext).Click();
+                ApplicationData.hearingPhone = new TextCopy.Clipboard().GetText();
```

Comment:
we're using ScenarioContext to share data between steps. I think we should use same approach here instead of ApplicationData

Reply:
changed